constants.h

```diff
@@ -6,11 +6,20 @@
 #define K_LEFT              6
 #define K_RIGHT             7
 #define K_UP                8
-#define K_DOWN              9
+#define K_DOWN              3
 #define K_FIRE              10
 
+// SNES Controller
+constexpr uint8_t DATA_CLOCK   = 11;
+constexpr uint8_t DATA_LATCH   = 12;
+constexpr uint8_t DATA_SERIAL  = 13;
+
+// Sound
+constexpr uint8_t SOUND_PIN   = 9; // do not change, belongs to used timer
+
 // GFX settings
 #define OPTIMIZE_SSD1306                // Optimizations for SSD1366 displays
+#define SNES_CONTROLLER
```

Comment:
Could you move this up next to SNES controller settings, and comment the line by default?

Reply:
Hi,

most of the additional fps comes from switching the i2c communication to non-blocking, so that the next frame can be calculated while data (1/3 of the image) is still being sent to the display.

> Could you move this up next to SNES controller settings, and comment the line so it´s disabled by default?

Yeah, no problem. I'll update the pull request then

Btw happy easter

Reply:
I've changed the SNES_CONTROLLER define and rebased to the latest master. Should be ready to pull now

Reply:
This is great. Thanks so much.